app/stores/ada/AdaWalletsStore.js

```diff
@@ -2,20 +2,15 @@
 // import { BigNumber } from 'bignumber.js';
 import { observable, action } from 'mobx';
 import WalletStore from '../base/WalletStore';
-import type { Features } from 'trezor-connect';
 import { matchRoute, buildRoute } from '../../utils/routing';
 import Request from '../lib/LocalizedRequest';
 import { ROUTES } from '../../routes-config';
 import type { WalletImportFromFileParams } from '../../actions/ada/wallets-actions';
 import type { ImportWalletFromFileResponse } from '../../api/ada/index';
 import type {
-  CreateTransactionResponse,
-  CreateWalletResponse,
-  DeleteWalletResponse,
-  GetWalletsResponse,
-  RestoreWalletResponse,
-  ConnectTrezorResponse,
-  GenerateWalletRecoveryPhraseResponse,
+  CreateTransactionResponse, CreateWalletResponse, DeleteWalletResponse,
```

Comment:
we should be consistent with the formatting. This doesn’t seem like what we do :-/

Reply:
If you see the `develop` branch:
https://github.com/Emurgo/yoroi-frontend/blob/f23f1e8e975161b1c41d2dcba157fd0609bc864c/app/stores/ada/AdaWalletsStore.js#L10

I made it `one import per line` because before i was using this store for Trezor (`ConnectTrezorResponse`),
since i moved everything to `TrezorConnetStore` I also undo `one import per line` because i am not longer changing anything in this file.

I think we should do overall refactoring in some another PR.

Reply:
good as it is